www/app/common/yourAlert.js

```diff
@@ -101,7 +101,78 @@
                     var api = 'mobile_api/api.php?action=notify&type=account&device_id=' + $scope.uuid + '&user_id=' + userid + '&email=' + email;
                     self.AllData(api);
                 };
-              
+                if ($stateParams.wid) {
+                    $ionicLoading.show({
+                        templateUrl: 'partials/modals/productPage/loading.html',
+                        scope: $scope
+
+                    });
+                    var api = 'mobile_api/api.php?action=notify&type=account&device_id=' + $scope.uuid + '&user_id=' + userid + '&email=' + email;
```

Comment:
can we use a seperate service for scope.uuid 
i.e to generate uuid rather than using device.uuid everywhere